src/java.compiler/share/classes/javax/tools/StandardJavaFileManager.java

```diff
@@ -262,10 +263,15 @@
      *     getJavaFileObjectsFromPaths({@linkplain java.util.Arrays#asList Arrays.asList}(paths))
      * </pre>
      *
+     * @implSpec
+     * The default implementation will only throw {@linkplain NullPointerException NullPointerException}
```

Comment:
`{@linkplain}` should only be used for natural-language text. 
Alternatively, it *may* work to use `{@linkplain NullPointerException {@code NullPointerException}}` but that seems overkill.